policy/modules/services/podman.te

```diff
@@ -28,6 +28,16 @@ application_domain(podman_conmon_t, podman_conmon_exec_t)
 type podman_conmon_user_t;
 application_domain(podman_conmon_user_t, podman_conmon_exec_t)
 
+# Ensure conmon runs in s0 so that it can talk to the container
+ifdef(`enable_mcs',`
+	range_transition podman_t podman_conmon_exec_t:process s0;
+	range_transition podman_user_t podman_conmon_exec_t:process s0;
+')
+ifdef(`enable_mls',`
+	range_transition podman_t podman_conmon_exec_t:process s0;
+	range_transition podman_user_t podman_conmon_exec_t:process s0;
+')
```

Comment:
These should not be part of the declaration block. Please move them.

Reply:
OK. I've moved these to the conmon block.

Reply:
The rules should go in the existing `podman_t` and `podman_user_t` blocks. The two enable_* blocks should go above the init_systemd block.